### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brandones**
from https://wiki.openmrs.org/display/projects/openmrs-esm-patient-chart

I also added some links and cleaned up the copy a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,217 @@
...
 
-More info on the [wiki][]
+openmrs-esm-patient-chart is a patient dashboard microfrontend for the 
+OpenMRS SPA. It provides a simple dashboard with cards detailing the 
+patient's information, such as vitals, demographic and relationships.
 
-[wiki]: https://wiki.openmrs.org/display/projects/openmrs-esm-patient-chart
+## How do I install this?
+
+Requirements: Java 8, Latest NodeJS & NPM, Latest Git, Latest OpenMRS
```
Comment (FlorianRappl) [inline README.md:13]: Why is Java 8 a requirement?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,217 @@
 [![Build Status](https://travis-ci.org/openmrs/openmrs-esm-patient-chart.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-esm-patient-chart)
 
-Openmrs ESM Patient Chart
-=========
+# Openmrs ESM Patient Chart
```
Comment (FlorianRappl) [inline README.md:3]: OpenMRS?

Comment (FlorianRappl) [reply README.md:3]: (not sure here - sometimes we use something like `openmrs-esm-patient-chart`, but here it seems we want it in title case)

Comment (brandones) [reply README.md:3]:
```suggestion
# OpenMRS ESM Patient Chart
```

File: README.md (hunk 1 above)
Comment (brandones) [reply README.md:13]: It isn't, except for OpenMRS itself. @PermissionError can correct me if I'm wrong.

**Pull request merged**